docs/mine-hnt/validators/wallet.mdx

```diff
@@ -30,6 +30,14 @@ At any time use ```-h``` or ```--help``` to get more help for a command.
 
 ```./helium-wallet create basic --network testnet``` 
 
+#### Create a Wallet in a non-interactive / DevOps way
+
```

Comment:
For security reasons, it is generally recommended that you type in your password every time. Storing the password in an environmental variable makes it very easy for the password to get compromised. Sometimes though, despite the security risks, you may want to interact with the CLI without need to input the password. In this case, using the environmental variable `HELIUM_WALLET_PASSWORD` enables this.

Reply:
The clarification is valid and explains why the variable exists, however I'd like to insist in specifying an example (like [this one](https://github.com/helium/docs/pull/218/files/4791a853daf3bccc8adc467a4ce93079a75e8a8f#diff-29853b474d408aac49fc4717bb1e6b78e0c1efc2d5b5b7c54c106a1926c962b1) so nobody is encouraged to use `export`.

Reply:
This is the proposal. As I mentioned earlier, please accept the suggestions if you agree. I don't want to push to your branch if I can avoid it.

Reply:
@lthiery Agree to the modifications!